TODO.md

```diff
@@ -0,0 +1,25 @@
+## 9/8
```

Comment:
I agree with this. I had just added these for myself in part - will remove, put them in issues, and think about how a roadmap doc should be outlined.